### PR TITLE
put back the config to write logs to the file instead of only stdout logs

### PR DESCRIPTION
**Pull request by paulyufan2**
**Reason for Change**:

This PR is to put back thye config to write to the file instead of only stdout logs

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

File: cni/network/plugin/main.go [hunk 1]
```diff
@@ -295,14 +296,21 @@ func main() {
...
 	}
 
-	loggerCfg := &log.Config{
+	log.SetName(name)
+	log.SetLevel(log.LevelInfo)
+	if err := log.SetTargetLogDirectory(log.TargetLogfile, ""); err != nil {
+		fmt.Printf("Failed to setup cni logging: %v\n", err)
+		return
+	}
+
```
Comment (tamilmani1989) [inline cni/network/plugin/main.go:305]: can we move this before `NewPlugin` api is called?

Comment (tamilmani1989) [inline cni/network/plugin/main.go:305]: call defer log.Close()

**Review by tamilmani1989 — CHANGES_REQUESTED** [review]

File: cni/network/plugin/main.go (hunk 1 above)
Comment (paulyufan2) [reply cni/network/plugin/main.go:305]: it was added on line 306

Comment (paulyufan2) [reply cni/network/plugin/main.go:305]: Hi Tamilmani, can you please explain more about this? This is already before NewPluigin(rootExecute()) is called.

**Comment by vipul-21** [comment]
@paulyufan2 make sure you run the `Azure Container Networking PR - submodules` pipeline. It is getting skipped as the criteria to trigger the pipeline didn't match.( Will change that in separate PR)

**Comment by paulyufan2** [comment]
> @paulyufan2 make sure you run the `Azure Container Networking PR - submodules` pipeline. It is getting skipped as the criteria to trigger the pipeline didn't match.( Will change that in separate PR: #2108 )

This is the submodule pipeline log:

File: cni/network/plugin/main.go [hunk 2]
```diff
@@ -15,9 +15,10 @@ import (
 	"github.com/Azure/azure-container-networking/aitelemetry"
 	"github.com/Azure/azure-container-networking/cni"
 	"github.com/Azure/azure-container-networking/cni/api"
-	"github.com/Azure/azure-container-networking/cni/log"
+	zaplog "github.com/Azure/azure-container-networking/cni/log"
```
Comment (ramiro-gamarra) [inline cni/network/plugin/main.go:18]: is this change needed? i'm not sure why the import alias is prefixed with `zap` (seems like an implementation detail). less changed lines if you just alias the new conflicting import?

Comment (paulyufan2) [reply cni/network/plugin/main.go:18]: Hi Ramiro, because we want to rename the import packet since they are all using "log." to avoid conflicts

File: cni/network/plugin/main.go (hunk 1 above)
Comment (tamilmani1989) [reply cni/network/plugin/main.go:305]: ok then its fine

**Review by tamilmani1989 — APPROVED** [review]
